ui/common/css/theme/_transp.scss

```diff
@@ -39,6 +39,13 @@ $spotlight-color-green: $c-color-choice-green;
 $spotlight-color-red: $c-color-choice-red;
 $spotlight-color-yellow: $c-color-choice-yellow;
 
+$spotlight-icon-color: $c-link;
+$spotlight-icon-color-black: $c-color-choice-black-dim;
```

Comment:
these are not required as there is an import from dark at top of file, and the values are identical.

Reply:
ah thanks :+1:

Reply:
will remove all the spotlight variables